resources/sdk/purecloudjavascript/templates/ApiClient.mustache

```diff
@@ -1163,6 +1194,61 @@ class ApiClient {
 		return url;
 	}
 
+    /**
+    	 * Builds http client.
+    	 * @returns {String} The http client default or the one that is injected.
+    	 */
+    	buildHttpClient() {
+    		if (this.httpClient) {
+    		    return this.httpClient;
+    		} else {
+    		    return this.defaultClient();
+    		}
+    	}
+
+        buildRequestOptions(url, method, headers, data, params) {
```

Comment:
Not critical. But the buildRequestOptions could be removed and its code moved to a RequestOptions constructor.